### ISSUE TITLE
Testnet Bitcoin balance is zero after successful txs

### ISSUE THREAD

**Issue by geoffwhittington** [renproject/send-crypto#16]
* **I'm submitting a ...**
[ ] bug report
[ ] feature request
[ ] question about the decisions made in the repository
[ X] question about how to use this project

* **Summary**
Sending test bitcoin to `moPDNpU3NSXEGXzZAwjugYxYutmYpwAgS8` shows up here: https://blockstream.info/testnet/address/moPDNpU3NSXEGXzZAwjugYxYutmYpwAgS8 however it is not showing up here: https://trest.bitcoin.com/v2/address/details/moPDNpU3NSXEGXzZAwjugYxYutmYpwAgS8

Do I have a misunderstanding of bitcoin testnet - can you help me clarify things? 

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)

**Comment by geoffwhittington** [comment]
It appears testnet bitcoin is no longer supported at bitcoin.com ? https://www.reddit.com/r/btc/comments/fyydwa/alternatives_to_restbitcoincom/

**Comment by iamnotstatic** [comment]
Yeah, you have to specify a network on the constructor while creating a new instance of CryptoAccount

```js
const account = new CryptoAccount(privateKey, {
    network: 'testnet',
});
````
 This should work.
